app/metrics/Monitors.scala

```diff
@@ -16,59 +16,13 @@
 
 package metrics
 
-import connectors.MessageConnector.EisSubmissionResult
-import models.Arrival
+import com.codahale.metrics.MetricRegistry
 import models.ChannelType
 import models.MessageType
 import models.SubmissionProcessingResult
-import models.response.ResponseArrival
-import play.api.libs.ws.WSResponse
-
-trait RequestMonitor[A] {
-  val name: String
-
-  val path: String         = s"$name.request"
-  val timer: Timer         = Timer(path)
-  val callCounter: Counter = Counter(path)
-
-  def completionCounter(result: A): Option[Counter] = None
-  val failureCounter: Counter                       = Counter(s"$path.failed")
-}
```

Comment:
A Dropwizard Metrics `Timer` is already a `Counter` of requests and a `Meter` of the rate of requests, so I have removed this request monitor abstraction. 

Also, the `grafana-dashboards` project expects metric names to follow a certain format - `<metric-name>` for the `DownstreamResponseTimesGraphBuilder` and `<metric-name>-success-counter` and `<metric-name>-failed-counter` in order to work with the `DownstreamResponsesPerMinuteGraphBuilder`